.github/workflows/tests.yml

```diff
@@ -225,11 +227,11 @@ jobs:
             python -c 'import PyQt5.QtCore' &&
             echo 'PyQt5 is available' ||
             echo 'PyQt5 is not available'
-          if [[ "${{ runner.os }}" != 'macOS' ]]; then
+          if [[ "${{ matrix.os }}" = ubuntu-20.04 ]]; then
```

Comment:
Why skip on `ubuntu-22.04` or `ubuntu-latest`?

Reply:
At the time, only 3.12 was running one of those so it coincided.